src/base/_defaults.scss

```diff
@@ -151,3 +151,28 @@ table {
   text-align: left;
   width: 100%;
 }
+
+/*
+ * 1. Removes default margin
+ * 2. Center content
+ * 3. Space between elements inside <figure>
+ */
+
+figure {
+  margin: 0; /* 1 */
+  text-align: center;
```

Comment:
I see now why this doesn't replace `margin-left: auto` and `margin-right: auto`. Let's move it back to `figcaption`, it made more sense there. Sorry for steering you wrong.